fontforgeexe/fontinfo.c

```diff
@@ -2657,7 +2657,7 @@ static int CheckNames(struct gfi_data *d) {
     buts[0] = _("_OK"); buts[1] = _("_Cancel"); buts[2]=NULL;
 
     if ( u_strlen(ufont)>63 ) {
-	ff_post_error(_("Bad Font Name"),_("A PostScript name should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
+	ff_post_error(_("Bad Font Name"),_("A Fontname should be ASCII\nand must not contain (){}[]<>%%/ or space\nand must be shorter than 63 characters"));
```

Comment:
Either "A font name" or just "FontName" (referencing the attribute).

Anyway, I don’t think this statement is true outside of Postscript.

Also I note that the current message says Postscript name because this rule applies to all Postscript names. Only the window title ought to display the individual attribute (see how the window titles differs across the different messages).